ddsketch/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "datadog-ddsketch"
+authors = ["Paul Le Grand des Cloizeaux <paul.legranddescloizeaux@datadoghq.com>"]
```

Comment:
You should add your name as well @VianneyRuhlmann

Reply:
I don't think we should be using the author attribute here at all, actually.